src/applications/gi/components/profile/CalculatorForm.jsx

```diff
@@ -571,15 +589,45 @@ class CalculatorForm extends React.Component {
       const errorMessageCheck =
         errorMessage !== '' ? errorMessage : inputs.beneficiaryZIPError;
 
-      amountInput = (
+      // Prod Flag for 19703
+      if (environment.isProduction() || !inputs.classesOutsideUS) {
+        // Prod Flag for 19703
```

Comment:
Accidental copy of the commment above?

Reply:
comment is for each `environment.isProduction()` usage, could have been reduced to one